src/webrequest.js

```diff
@@ -55,6 +55,10 @@ function onBeforeRequest(details){
     recordFrame(details.tabId, details.frameId, details.parentFrameId, details.url);
   }
 
+  if (type == "ping" && details.tabId < 0 ){
```

Comment:
So if that is the case should we remove `details.tabId < 0` from the `if` clause?